pyDeltaRCM/bmi_delta.py

```diff
@@ -47,7 +144,40 @@ def initialize(self, filename='deltaRCM.yaml'):
             Path to name of input file.
         """
 
-        self._delta = pyDeltaRCM(filename)
+        input_file_vars = dict()
+
+        # Open and access yaml file --> put in dictionaries
+        # only access the user input file if provided.
+
+        if filename != 'deltaRCM.yaml' and os.path.exists(filename):
+            user_file = open(filename, mode = 'r')
+            user_dict = yaml.load(user_file, Loader = yaml.FullLoader)
+            user_file.close()
+        else:
+            print('The specified input file could not be found. Using default values...')
```

Comment:
Should probably use a warning here instead? I'm fine with this for now though, we can clean up any other  `print` warnings with a later PR. Maybe you could file an issue to remind us?

Reply:
Will do.